src/homebridge-vantage-static.ts

```diff
@@ -64,10 +68,19 @@ class VantageStaticPlatform implements StaticPlatformPlugin {
     this.log.info("Done initializing homebridge vantage platform");
   }
 
-  vidToName(vid: string): string {
-    const mappingsKeys = Object.keys(this.vidNameMapping);
-    if (mappingsKeys.length !== 0 && mappingsKeys.includes(vid)) {
-      return this.vidNameMapping[vid];
+  vidToName(vid: string): string | undefined {
+    const mappingsKeys = Object.keys(this.vidMapping);
+    if (mappingsKeys.length !== 0 && mappingsKeys.includes(vid) && this.vidMapping[vid]["Name"]) {
+      return this.vidMapping[vid]["Name"];
+    } else {
+      return "";
+    }
```

Comment:
maybe warn or raise in this case? what happens if `""` is returned?

Reply:
Its a valid outcome. If the `Name` isn't specified in the mappings then I want to return `""`, or maybe I should return `undefined`. 
If I get `""` I just take the default name from the configuration.